petabvis/bar_row.py

```diff
@@ -54,7 +54,7 @@ def get_sd(self):
         """
         Return the standard deviation of the y-values that should be plotted.
         Returns:
-            sd: The standard deviation
+            sd: The standard deviation.
```

Comment:
```suggestion
            The standard deviation.
```

General comment: For return values, the variable name is not really useful in the docstring. You can just drop that.